openmc/data/data.py

```diff
@@ -11,106 +11,106 @@
 # of the elements 2013 (IUPAC Technical Report)", Pure. Appl. Chem. 88 (3),
 # pp. 293-306 (2013). The "representative isotopic abundance" values from
 # column 9 are used except where an interval is given, in which case the
-# "best measurement" is used.
+# average of the interval is used.
 # Note that the abundances are given as atomic fractions!
 NATURAL_ABUNDANCE = {
-    'H1': 0.99984426, 'H2': 0.00015574, 'He3': 0.000002,
-    'He4': 0.999998, 'Li6': 0.07589, 'Li7': 0.92411,
-    'Be9': 1.0, 'B10': 0.1982, 'B11': 0.8018,
```

Comment:
It looks like the leading whitespace changed on all these lines -- can you please revert it to how it was before?